deploy/helm_charts/README.md

```diff
@@ -36,3 +36,47 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
     --set-file kgStoreConfig.bigqueryVersion=deploy/storage/bigquery.version \
     --set-file kgStoreConfig.bigtableImportGroupsVersion=deploy/storage/bigtable_import_groups.version
 ```
+
+## Releasing a helm chart
+
+Releasing a helm chart packages and uploads a helm chart into [AR(Artifact Registry)](https://cloud.google.com/artifact-registry/docs/overview) as a versioned artifact. Once released, the chart will be accessible by the public, given that the access to AR repository is configured.
+
+### Prerequisites
+
+Make sure that the repository exists with the following gcloud command.
```

Comment:
Curious when would a developer need to create a new repo?

Reply:
Deleted that comment, it was a one time setup and it should always exist.